Tella/Scenes/HomeView/ViewModel/HomeViewModel.swift

```diff
@@ -28,16 +28,24 @@ class HomeViewModel: ObservableObject {
     }
     
     func getServersList() {
-        
-        self.appModel.vaultManager.tellaData.servers.sink { result in
             
-        } receiveValue: { serverArray in
-            self.serverDataItemArray.removeAll()
-            if !serverArray.isEmpty {
-                self.serverDataItemArray.append(ServerDataItem(servers: serverArray, serverType: .tellaUpload))
-            }
-        }.store(in: &subscribers)
-    }
+            self.appModel.vaultManager.tellaData.servers.sink { result in
+                
+            } receiveValue: { serverArray in
+                self.serverDataItemArray.removeAll()
+                if !serverArray.isEmpty {
+                    // here i group all the tella servers in one array and the third party services in diferents arrays
+                    let thirdPartyConnections = serverArray.filter { mapServerTypeFromInt($0.serverType) != .tella }
+                    let tellaUploadServers = serverArray.filter { mapServerTypeFromInt($0.serverType) == .tella }
```

Comment:
Could you remove `mapServerTypeFromInt` ?

Reply:
we are using `mapServerTypeFromInt` to parse the value that we save in the database to the serverType we use in the app.

We are storing in the DB an integer to reference the connection type and then when we fetch the table we use that method to parse it. What alternative do you think could replace this method?